test/extended/operators/check_crd_status.go

```diff
@@ -0,0 +1,125 @@
+package operators
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+	"k8s.io/kube-openapi/pkg/util/sets"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	apiextensionsclientset "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// hasStatus returns true of the given crdItem has a Status defined.
+func hasStatus(crdItem apiextensionsv1.CustomResourceDefinition) bool {
```

Comment:
this looks confused.  You want to know if the schema in the CRD spec has a status element defined.